core/src/main/scala/pink/cozydev/lucille/QueryParser.scala

```diff
@@ -205,10 +225,11 @@ private object Parser {
         rangeQuery,
         fuzzyT,
         prefixT,
+        minimumMatchQ(r),
+        boostQ(r),
```

Comment:
`minimumMatchQ` got moved here because `boostQ` must be after `minimumMatchQ` AND before `termQ`